cli/src/geojson_to_osmosis.rs

```diff
@@ -0,0 +1,16 @@
+use anyhow::Result;
+
+use geom::LonLat;
+
+pub fn run(path: String) -> Result<()> {
```

Comment:
I changed to take a file as a path, instead of read from STDIN. Much easier to use this as a library call from `one_step_import.rs`.